CHANGELOG.md

```diff
@@ -15,8 +15,15 @@
 * [ENHANCEMENT] Make trace combination/compaction more efficient [#1291](https://github.com/grafana/tempo/pull/1291) (@mdisibio)
 * [ENHANCEMENT] Add Content-Type headers to query-frontend paths [#1306](https://github.com/grafana/tempo/pull/1306) (@wperron)
 * [ENHANCEMENT] Partially persist traces that exceed `max_bytes_per_trace` during compaction [#1317](https://github.com/grafana/tempo/pull/1317) (@joe-elliott)
+  ```
+  query_frontend:
+  search:
+    hedge_requests_at: 5s
+    hedge_requests_up_to: 3
+  ```
```

Comment:
Is this intentional? It seems to be copied in the wrong place.

Reply:
wups, somehow my changelog message got dropped. fixing.